src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -926,6 +927,14 @@ public String getToolName() {
         return String.format("%s %s", getToolkitShortName(), getClass().getSimpleName());
     }
 
+    /**
+     * Returns the list of intervals to iterate, either limited to the user-supplied intervals or the entire reference genome if none were specified.
+     * If no reference was supplied, null is returned
+     */
+    public List<SimpleInterval> getEffectiveIntervals() {
```

Comment:
The method name you chose here is a bit confusing -- @cmnbroad can you suggest a better name in your final review? Perhaps this should be `getTraversalIntervals()`, and we could rename the field `intervalsForTraversal` to the somewhat more accurate `userIntervals`?

Reply:
That would seem reasonable.  hasIntervals() should probably also get renamed hasUserSuppliedIntervals() as well?